### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by nforro**

### CODE REVIEW

File: .github/workflows/pypi-publish.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        # https://pypa-build.readthedocs.io
+        run: |
+          python -m pip install build
+          python -m build --sdist --wheel
+
+      - name: Publish 📦 to PyPI
+        # https://github.com/pypa/gh-action-pypi-publish
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
```
Comment (FrNecas) [inline .github/workflows/pypi-publish.yml:28]: I assume the secret is set on org level, but we should verify that just to make sure.

**Review by FrNecas — APPROVED** [review]

File: .github/workflows/pypi-publish.yml (hunk 1 above)
Comment (nforro) [reply .github/workflows/pypi-publish.yml:28]: As far as I can tell, it's not, I actually set my own token on repo level.

Comment (nforro) [reply .github/workflows/pypi-publish.yml:28]: It's a temporary one anyway, as the project on PyPI doesn't exist yet.

Comment (FrNecas) [reply .github/workflows/pypi-publish.yml:28]: Oh, I remember now, the token is not tied to an account but rather a project IIRC. All good then :)

**Pull request merged**